tcmur_cmd_handler.c

```diff
@@ -2269,6 +2371,15 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 		break;
 	}
 
+	rcmd_state = calloc(1, sizeof(*rcmd_state));
+	if (!rcmd_state) {
+                tcmu_dev_err(dev, "Failed to calloc memory for tcmur_cmd_state!\n");
+                ret = TCMU_STS_NO_RESOURCE;
+                goto untrack;
+        }
+
```

Comment:
The same here with the spaces issue.

Reply:
Will fix the spaces issues above. Thanks.